hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
+  (for* [k (+ (, key1) keys)]
+    (setv thing (get thing k)))
+  thing)
```

Comment:
Oh, I thought you meant some version using `reduce`. But why use the `if`?

    (defn get [coll key1 &rest keys]
      (setv coll (get coll key1))
      (for* [k keys]
        (setv coll (get coll k)))
      coll)